firefox-ios/Storage/DefaultSuggestedSites.swift

```diff
@@ -129,8 +122,14 @@ open class DefaultSuggestedSites {
                 trackingId: 705,
                 // FXIOS-11064 We need a higher quality favicon link
                 faviconResource: .remoteURL(url: URL(string: "https://corporate.jd.com/favicon.ico")!)
+            ),
+            Site.createSuggestedSite(
+                url: "http://douyin.com",
+                title: "抖音",
+                trackingId: 706,
+                faviconResource: .remoteURL(url: URL(string: "https://lf-douyin-pc-web.douyinstatic.com/obj/douyin-pc-web/2025_0313_logo.png")!)
             )
-         ],
+        ],
```

Comment:
before after:

<img width="589" height="1280" alt="Simulator Screenshot - iPhone 17 Pro Max - 2025-11-25 at 22 15 05 Large" src="https://github.com/user-attachments/assets/57d4750e-068c-4842-af4d-c62625fd2752" />

<img width="589" height="1280" alt="Simulator Screenshot - iPhone 17 Pro Max - 2025-11-25 at 21 59 24 Large" src="https://github.com/user-attachments/assets/923f50e8-914d-42f7-8a33-04f118d6fec1" />

Reply:
If the addition of anything new is not uncontroversial and would delay this having to bikeshed about the content, let me know and I'll drop the new tile here.

Primarily I wanted to get rid of the broken link on the first tile. Thus happy to do just a removal without any new addition and keeping only 5 tiles.